pkg/servicemonitor/servicemonitor.go

```diff
@@ -53,6 +54,26 @@ func (u *ServiceMonitor) UpdateServiceMonitorDeployment(template monitoringv1.Se
 	return nil
 }
 
+// Creates or Updates Service Monitor Deployment according to the template if enable of the hypershift
+func (u *ServiceMonitor) Hypershift_UpdateServiceMonitorDeployment(template rhobsv1.ServiceMonitor) error {
```

Comment:
```suggestion
func (u *ServiceMonitor) HypershiftUpdateServiceMonitorDeployment(template rhobsv1.ServiceMonitor) error {
```

Reply:
Is it really depending on hypershift? Or is it more for users who use OO and RHOBS CRDs instead of CMO?